src/server/scripts/Northrend/Ulduar/Ulduar/boss_auriaya.cpp

```diff
@@ -186,26 +186,17 @@ class boss_auriaya : public CreatureScript
 
             summons.DoZoneInCombat(NPC_SANCTUM_SENTRY);
 
-            me->Yell("Some things are better left alone!", LANG_UNIVERSAL);
-            me->PlayDirectSound(SOUND_AGGRO);
+            Talk(SAY_AGGRO);
             me->setActive(true);
         }
 
-        void KilledUnit(Unit*  /*victim*/) override
+        void KilledUnit(Unit* victim) override
         {
             if (urand(0, 2))
                 return;
 
-            if (urand(0, 1))
-            {
-                me->Yell("The secret dies with you!", LANG_UNIVERSAL);
-                me->PlayDirectSound(SOUND_SLAY1);
-            }
-            else
-            {
-                me->Yell("There is no escape!", LANG_UNIVERSAL);
-                me->PlayDirectSound(SOUND_SLAY2);
-            }
+            if (victim->GetTypeId() == TYPEID_PLAYER)
```

Comment:
Is this check neccessary?

Reply:
Without that check, wouldn't she talk every time she kills anything (pets, etc.)?

Reply:
I am want to say no, as we use it so many other places and K never heard of it. Can you test it please?

Reply:
@Kitzunu 
After a quick check on other bosses in raids all over Northrend it appears that all of them have use variation of:
 >void KilledUnit(Unit* who) override
            if (who->GetTypeId() != TYPEID_PLAYER)
                return;

Should be there to prevent unintended boss speech when killing anything, but a player, unless stated otherwise.

Without this check -  yells on anything killed (disabled the urand for the test)

https://user-images.githubusercontent.com/46330494/167317626-8b7dedfa-7703-4c06-9bad-ac47f32e6aea.mp4


With the check -  only yells on player kill (disabled the urand for the test)


https://user-images.githubusercontent.com/46330494/167318039-371f3a7b-657b-4041-910f-b87a990d96d9.mp4